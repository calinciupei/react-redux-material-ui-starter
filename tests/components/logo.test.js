import React from 'react';
import { shallow } from 'enzyme';

import Logo from '../../src/components/logo/logo.component';

let context;

const createComponent = () => shallow(<Logo />);

describe('<Logo />', () => {
  context = createComponent();

  it('should render correctly', () => {
    expect(context).toMatchSnapshot();
  });
});
