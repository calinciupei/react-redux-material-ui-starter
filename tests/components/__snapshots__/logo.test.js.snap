// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should render correctly 1`] = `
<div
  className="logo"
>
  <span
    className="logo__text"
  >
    Logo
  </span>
</div>
`;
